### PR TITLE
[incubator/zookeeper] Rename containers and values

### PR DESCRIPTION
**Pull request by h0tbird**
I have also added myself as maintainer.

### CODE REVIEW

File: incubator/zookeeper/templates/statefulset.yaml [hunk 1]
```diff
@@ -9,14 +9,14 @@ metadata:
     heritage: {{ .Release.Service | quote }}
 spec:
   serviceName: {{ template "fullname" . }}-headless
-  replicas: {{ .Values.Servers }}
+  replicas: {{ default 3 .Values.servers }}
```
Comment (unguiculus) [inline incubator/zookeeper/templates/statefulset.yaml]: `default 3` is unnecessary.

File: incubator/zookeeper/Chart.yaml [hunk 2]
```diff
@@ -11,3 +11,5 @@ maintainers:
     email: lachlan@deis.com
   - name: Kenneth Owens
     email: owensk@google.com
+  - name: Marc Villacorta
```
Comment (unguiculus) [inline incubator/zookeeper/Chart.yaml]: Are you willing to contribute more and to maintain the chart in the future, since you added yourself here?

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for this update. Much appreciated.

**Comment by unguiculus** [comment]
Marking this as stale. Please respond to comments.

**Comment by unguiculus** [comment]
Ping @h0tbird

File: incubator/zookeeper/Chart.yaml (hunk 2 above)
Comment (h0tbird) [reply incubator/zookeeper/Chart.yaml]: I plan to add an optional Prometheus exporter but I can not commit to much more.

Comment (unguiculus) [reply incubator/zookeeper/Chart.yaml]: The don't add yourself as owner.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**